webpack.config.ts

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  entry: './src/cli.ts',
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/,
+      },
+      {
+        test: /\.node$/,
+        loader: 'node-loader',
+      },
+      {
+        test: path.resolve(__dirname, 'node_modules/simple-websocket/server.js'),
+        use: 'null-loader',
+      },
+    ],
+  },
+  resolve: {
+    extensions: [ '.tsx', '.ts', '.js' ],
+  },
+  
+  node: {
+    __dirname: false,
+    fs: 'empty',
+  },
+
+  externals: [
```

Comment:
do we need externals considering we have null-loader?

Reply:
need to 2-check

Reply:
works fine without